drivers/led/sn32f2xx.c

```diff
@@ -263,13 +379,15 @@ static void update_pwm_channels(PWMDriver *pwmp) {
     // Disable LED output before scanning the key matrix
     if (current_key_row < ROWS_PER_HAND) {
         shared_matrix_rgb_disable_output();
+#   if defined(SHARED_MATRIX)
         shared_matrix_scan_keys(shared_matrix, current_key_row, last_key_row);
+#   endif // SHARED_MATRIX
     }
     bool enable_pwm_output = false;
     for (uint8_t current_key_col = 0; current_key_col < SN32_RGB_MATRIX_COLS; current_key_col++) {
         uint8_t led_index = g_led_config.matrix_co[current_key_row][current_key_col];
 #    if (SN32_PWM_CONTROL == SOFTWARE_PWM)
-        if (led_index >= SN32F24XB_LED_COUNT) continue;
+        if (led_index >= SN32F2XX_LED_COUNT) continue;
```

Comment:
For me this line is necessary also for hardware PWM.
If this is not used the last keys in cols which have less then the standard amount of keys.  (e.g. just 4 than normally 5) will light up dimly.